lib/udap_security_test_kit/client_suite/token_verification.rb

```diff
@@ -1,59 +1,34 @@
-require_relative '../tags'
-require_relative '../urls'
-require_relative '../endpoints/mock_udap_server'
-
 module UDAPSecurityTestKit
-  class UDAPClientTokenRequestVerification < Inferno::Test
-    include URLs
-
-    id :udap_client_token_request_verification
-    title 'Verify UDAP Token Requests'
-    description %(
-      Check that UDAP token requests are conformant.
-    )
-
-    output :udap_demonstrated
-    output :udap_tokens
-
-    run do
-      load_tagged_requests(REGISTRATION_TAG, UDAP_TAG)
-      output udap_demonstrated: requests.present? ? 'Yes' : 'No'
-      omit_if requests.blank?, 'UDAP Authentication not demonstrated as a part of this test session.'
-      registration_request = requests.last
-      registration_assertion = MockUDAPServer.parsed_request_body(registration_request)['software_statement']
+  module TokenVerification
```

Comment:
This seems to be doing a lot more than verifying tokens.

Reply:
updated name to include "request" in [92fd83e](https://github.com/inferno-framework/udap-security-test-kit/pull/32/commits/92fd83e7e84e95668201a0abcd81af7599424286)